srml/system/src/lib.rs

```diff
@@ -105,6 +105,16 @@ pub trait OnNewAccount<AccountId> {
 impl<AccountId> OnNewAccount<AccountId> for () {
 	fn on_new_account(_who: &AccountId) {}
 }
+impl<
```

Comment:
Please do as its done here: https://github.com/paritytech/substrate/blob/master/srml/timestamp/src/lib.rs#L200

Reply:
Should I also update this then?
https://github.com/paritytech/substrate/blob/master/srml/support/src/traits.rs#L73

Reply:
Yeah